educator/database/schema.sql

```diff
@@ -18,18 +12,17 @@ create table if not exists courses (
     -- Sqlite3 will force ascending primary key to be non-null, replacing
     --  null with autoincremented key (while sqlite2 won't).
     --
-    id    INTEGER    PRIMARY KEY AUTOINCREMENT,
-    desc  TEXT     not null
+    id     BIGSERIAL  PRIMARY KEY,
+    "desc" TEXT       not null
 );
 
 -- Creating 'subjects' table.
 --
 create table if not exists subjects (
-    id          INTEGER  not null,
-    course__id  INTEGER  not null,
-    desc        TEXT     not null,
+    id          BIGSERIAL  PRIMARY KEY,
+    course__id  BIGINT     not null,
```

Comment:
Just in case, I've heard those scaring stories about people running out of `int32`, and @goodsoft advised so.
Regarding interpreting as unsigned - fair point actually, that would be more convenient.

Reply:
Using `Word64` does not work unfortunately, Postgres library already defines an instance which serializes `Word64` to `BIGINT` and that instance panics on overflows.
I can introduce a newtype for ids and define serialization sanely if that makes sense.